tasks-by-users/AbbasShaikh0304/p1/About-Us.html

```diff
@@ -0,0 +1,13 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>We sell the books at most affordable ranges.<br> Online Book store is an online web application where the customer can purchase books online. Through a web browser the customers can search for a book by its title or author, later can add to the shopping cart and finally purchase using credit card.<br>
+        <a href="/index.html">Back</a>
```

Comment:
indentation ? add prettier in VSCode.. and do necessary setting

Reply:
Gotcha 👍